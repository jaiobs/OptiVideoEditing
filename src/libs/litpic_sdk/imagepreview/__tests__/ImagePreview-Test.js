import React from 'react';
import renderer from 'react-test-renderer';
import ImagePreivew from '../ImagePreview';

describe("ImagePreview snapshot test",() => {
    it("It should render ImagePreview Correctly", () =>{
        const tree = renderer.create(
            <ImagePreivew></ImagePreivew>
        ).toJSON();
        expect(tree).toMatchSnapshot();
    })
});