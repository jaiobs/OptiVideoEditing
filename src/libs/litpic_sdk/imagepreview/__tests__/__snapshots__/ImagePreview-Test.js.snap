// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImagePreview snapshot test It should render ImagePreview Correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <imagePreviewAndroid />
</View>
`;
